test/Microsoft.Framework.ConfigurationModel.Test.Common/IConfigurationSourceExtensions.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.Framework.ConfigurationModel
+{
+    public static class IConfigurationSourceExtensions
```

Comment:
It is used just in tests for backward compat. Otherwise, I would have had to rewrite most tests to use `TryGet` which has an out param. Before, they were accessing directly the `Data` dictionary. I will rename the class to match the guidelines. 

It is recommended to use `TryGet` in product code because the setting might not be there